### ISSUE TITLE
Simplify Center Template Navigation

### ISSUE THREAD

**Issue by joewhitsitt** [ITS-UofIowa/education.uiowa.edu#47]
Related to #46 

Since the main site's navigation will be changing and can be edited by Education, it would be ideal to simplify the navigation on ICATER, etc. so that menu changes on these sites don't need to happen too often.

**Comment by joewhitsitt** [comment]
Once #46 is approved, I assume I would style it the same but just have a "Back to education.uiowa.edu" link in the header instead of a copy of the full menu. Is this correct?

**Comment by joewhitsitt** [comment]
Education will provide the design for the header to implement on sub sites.

**Comment by joewhitsitt** [comment]
It was decided this would be handle after launch of 1.3.0

**Comment by joewhitsitt** [comment]
icater is going away. this is no longer needed
